Auto rest

yanhuang1990 opened the pull request.

anthony0t committed:
Added 'code' and 'error' in workflow run, and 'error' in workflow run output parameter.

yugangw-msft committed:
Merge pull request #1347 from naveedaz/AutoRest

SDK Code gen from auto-generated swagger.json

Commit:
sync from master

Commit:
Merge branch 'master' into AutoRest

Conflicts:
	src/Authentication/Common.Authentication.Tests/Mocks/MockClientFactory.cs
	src/Authentication/Common.Authentication/Microsoft.Azure.Common.Authentication.nuget.proj
	src/Authentication/Common.Authentication/Properties/AssemblyInfo.cs
	src/ResourceManagement/Compute/Compute.Tests/ScenarioTests/VMImageTests.cs
	src/ResourceManagement/Compute/Compute.Tests/ScenarioTests/VMTestBase.cs
	src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.VMDataDiskTests/TestVMDataDiskScenario.json
	src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.VMImagesTests/TestVMImageGet.json
	src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.VMImagesTests/TestVMImageListFilters.json
	src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.VMImagesTests/TestVMImageListNoFilter.json
	src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.VMImagesTests/TestVMImageListOffers.json
	src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.VMImagesTests/TestVMImageListPublishers.json
	src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.VMImagesTests/TestVMImageListSkus.json
	src/ResourceManagement/Compute/Compute.Tests/SessionRecords/Compute.Tests.VMScenarioTests/TestVMScenarioOperations.json
	src/ResourceManagement/Compute/ComputeManagement/Generated/ComputeManagementClient.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/ComputeManagementClientExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IComputeManagementClient.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IUsageOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/AdditionalUnattendContent.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ApiError.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ApiErrorBase.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/AvailabilitySet.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/CachingTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ComponentNames.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ComputeOperationStatus.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DataDisk.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DataDiskImage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DiskCreateOptionTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DiskInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/HardwareProfile.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ImageReference.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/InnerError.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/InstanceViewStatus.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/LinuxConfiguration.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/NetworkInterfaceReference.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/NetworkProfile.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OSDisk.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OSDiskImage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OSProfile.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OperatingSystemTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/PassNames.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/Plan.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ProtocolTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/PurchasePlan.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/SettingNames.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/SshConfiguration.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/SshPublicKey.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/StatusLevelTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/StorageProfile.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/Usage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/UsageName.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/UsageUnit.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VaultCertificate.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VaultSecretGroup.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualHardDisk.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachine.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineAgentInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineCaptureParameters.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineExtension.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineExtensionHandlerInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineExtensionImage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineExtensionInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineImage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineImageResource.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineSize.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineSizeTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/WinRMConfiguration.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/WinRMListener.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/WindowsConfiguration.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/UsageOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/UsageOperationsExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Microsoft.Azure.Management.Compute.nuget.proj
	src/ResourceManagement/Compute/ComputeManagement/Properties/AssemblyInfo.cs
	src/ResourceManagement/Network/Network.Tests/SessionRecords/Networks.Tests.NetworkInterfaceTests/NetworkInterfaceApiTest.json
	src/ResourceManagement/Network/Network.Tests/SessionRecords/Networks.Tests.NetworkSecurityGroupTests/NetworkSecurityGroupApiTest.json
	src/ResourceManagement/Network/Network.Tests/SessionRecords/Networks.Tests.NetworkSecurityGroupTests/NetworkSecurityGroupWithRulesApiTest.json
	src/ResourceManagement/Network/Network.Tests/SessionRecords/Networks.Tests.SecurityRuleTests/SecurityRuleWithRulesApiTest.json
	src/ResourceManagement/Network/Network.Tests/Tests/ApplicationGatewayTests.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/ApplicationGatewayOperations.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/INetworkResourceProviderClient.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/Models/ApplicationGateway.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/Models/ApplicationGatewayBackendAddressPool.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/Models/ApplicationGatewayFrontendIpConfiguration.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/Models/ApplicationGatewayHttpListener.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/Models/ApplicationGatewayIpConfiguration.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/Models/NetworkInterface.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/Models/Subnet.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/NetworkInterfaceOperations.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/NetworkResourceProviderClient.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/SubnetOperations.cs
	src/ResourceManagement/Network/NetworkManagement/Generated/VirtualNetworkOperations.cs
	src/ResourceManagement/Network/NetworkManagement/Microsoft.Azure.Management.Network.nuspec
	src/ResourceManagement/Network/NetworkManagement/packages.config
	src/ResourceManagement/Resource/ResourceManagement/Generated/IResourceManagementClient.cs
	src/ResourceManagement/Resource/ResourceManagement/Generated/Models/DeploymentMode.cs
	src/ResourceManagement/Resource/ResourceManagement/Generated/ResourceManagementClient.cs
	src/ResourceManagement/Resource/Resources.Tests/ScenarioTests/ProviderTests.ScenarioTests.cs

Commit:
Merge master into AutoRest and switch branches

Commit:
Switch to AutoRest branch

Commit:
Bring files in line with AutoRest

copy/paste from official autoRest branch (faulty merge)

Commit:
Bringing branch in line with AutoRest

Commit:
Bring branch in line with AutoRest

Commit:
Missed these files due to ignore rules

Commit:
revert gitignore file

Needed to change it in order to commit files that are now ignored.

Commit:
Re-add changes for Kona/ADL

huangpf committed:
Merge pull request #71 from Azure/AutoRest

Auto rest

huangpf committed:
+generate.cmd

yugangw-msft committed:
disable parallel running as test fx does not support it

amarzavery committed:
Merge pull request #1359 from yugangw-msft/para

Address web site test flakiness

anthony0t committed:
Fixed build failure caused by changes in dependencies

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
Rev the package version

This is necessary to get the build to use my updated package instead of
the official one.

markcowl committed:
Merge pull request #1364 from tonytang-microsoft-com/AutoRest

Fixed build failure caused by changes in dependencies

Commit:
Add Resource Manager models

Commit:
Rename AzureProfile to AzureSMProfile

yugangw-msft committed:
get VM capture work

Commit:
Remove Supported Mode property from AzureSubscription

Commit:
Remove CSM subscription logic

Commit:
Extract out ProfileClient related code

ogail committed:
Merge pull request #1 from ogail/rmv

Remove Supported Mode property from AzureSubscription

Commit:
Remove CSM and RDFE management clients code

Commit:
Merge branch 'AutoRest' of github.com:ogail/azure-sdk-for-net into AutoRest

Commit:
Address CR

Commit:
Final pass of CR

Commit:
Update package version

ogail committed:
Merge pull request #1371 from ogail/AutoRest

Add Resource Manager models

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

huangpf committed:
Merge pull request #75 from Azure/AutoRest

Auto rest

huangpf committed:
Merge branch 'vm' of https://github.com/yugangw-msft/azure-sdk-for-net into autorest

Conflicts:
	src/ResourceManagement/Compute/ComputeManagement/Generated/AvailabilitySetsOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/AvailabilitySetsOperationsExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/ComputeManagementClient.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/ComputeManagementClientExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IAvailabilitySetsOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IComputeManagementClient.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IUsageOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IVirtualMachineExtensionImagesOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IVirtualMachineExtensionsOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IVirtualMachineImagesOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IVirtualMachineSizesOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/IVirtualMachinesOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/AdditionalUnattendContent.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ApiError.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ApiErrorBase.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/AvailabilitySet.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/AvailabilitySetListResult.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/CachingTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ComponentNames.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ComputeLongRunningOperationProperties.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ComputeLongRunningOperationResult.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ComputeOperationStatus.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DataDisk.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DataDiskImage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DeleteOperationResult.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DiskCreateOptionTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/DiskInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/HardwareProfile.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ImageReference.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/InnerError.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/InstanceViewStatus.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/LinuxConfiguration.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ListUsagesResult.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/NetworkInterfaceReference.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/NetworkProfile.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OSDisk.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OSDiskImage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OSProfile.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OperatingSystemTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/OperationStatus.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/PassNames.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/Plan.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/ProtocolTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/PurchasePlan.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/SettingNames.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/SshConfiguration.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/SshPublicKey.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/StatusLevelTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/StorageProfile.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/Usage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/UsageName.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/UsageUnit.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VaultCertificate.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VaultSecretGroup.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualHardDisk.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachine.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineAgentInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineCaptureParameters.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineExtension.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineExtensionHandlerInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineExtensionImage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineExtensionInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineImage.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineImageResource.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineInstanceView.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineSize.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineSizeListResult.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/VirtualMachineSizeTypes.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/WinRMConfiguration.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/WinRMListener.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/Models/WindowsConfiguration.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/UsageOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/UsageOperationsExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineExtensionImagesOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineExtensionImagesOperationsExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineExtensionsOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineExtensionsOperationsExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineImagesOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineImagesOperationsExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineSizesOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachineSizesOperationsExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachinesOperations.cs
	src/ResourceManagement/Compute/ComputeManagement/Generated/VirtualMachinesOperationsExtensions.cs
	src/ResourceManagement/Compute/ComputeManagement/generate.cmd

huangpf committed:
Update Recorded Data

bitsforthought committed:
Search SDK: Merging recent SDK changes from master to AutoRest

Used git cherry-pick to move these recent PRs from the master branch to the
AutoRest branch:

https://github.com/Azure/azure-sdk-for-net/pull/1384
https://github.com/Azure/azure-sdk-for-net/pull/1378

ogail committed:
Merge pull request #1389 from brjohnstmsft/searchmastermerge-pr

Search SDK: Merging recent SDK changes from master to AutoRest

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

huangpf committed:
Merge pull request #78 from Azure/AutoRest

Auto rest

markcowl committed:
Merge pull request #1372 from tonytang-microsoft-com/AutoRest

Added 'code' and 'error' in workflow run, and 'error' in workflow run…

anthony0t committed:
FW: Changed the version to 0.1.0-preview

yugangw-msft committed:
do not update assembly version

Commit:
Merge pull request #1405 from yugangw-msft/noupdate

do not update assembly version

yugangw-msft committed:
Merge pull request #1406 from tonytang-microsoft-com/AutoRest

FW: Changed the version to 0.1.0-preview

bitsforthought committed:
Search SDK: Fixing incorrect URL in Count docs

Since this is such a minor change, we're not updating the Nuspec
version. We'll ship this change with the next feature we add.

ogail committed:
Merge pull request #1407 from brjohnstmsft/docfix-pr

Search SDK: Fixing incorrect URL in Count docs

huangpf committed:
Merge pull request #84 from Azure/AutoRest

Auto rest

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
Updating my private version of the auth framework

markcowl committed:
Fixing test playback issue

markcowl committed:
Fixing test playback issue

markcowl committed:
Merge branch 'playback' of github.com:markcowl/azure-sdk-for-net into playback

markcowl committed:
removing unneeded files

huangpf committed:
Merge pull request #87 from markcowl/playback

Fixing test issue

huangpf committed:
Fix Tests

Commit:
Revving the version

1.1.5 exists on nuget.org, which is causing CI job failures since it is
picking up the wrong version.

x10shun committed:
Interim check-in. Updating client and tests

x10shun committed:
Merge remote-tracking branch 'upstream/AutoRest' into AutoRest

x10shun committed:
Fixed test cases.

huangpf committed:
Skip test assertion, pending on Yugang's fix in the common library

huangpf committed:
Update ClientRuntime Azure

huangpf committed:
Update

huangpf committed:
address CR

stankovski committed:
Merge pull request #1416 from huangpf/autorest

AzureRT S54 PR #1 - Compute Test Fix

Commit:
Updates for the new official name for kona

stankovski committed:
Merge pull request #1421 from naveedaz/AutoRest

Latest swagger json changes. Fixed and enabled more test cases

huangpf committed:
Merge pull request #94 from Azure/AutoRest

Auto rest

huangpf committed:
Update Versions

huangpf committed:
Update Compute Version

huangpf committed:
move

huangpf committed:
Update

Commit:
Exposed TokenCache and upgraded Authentication to version 1.2.0

stankovski committed:
Merge pull request #1430 from hovsepm/AutoRest

Exposed TokenCache and upgraded Authentication to version 1.2.0

Commit:
Moved TokenCache to Context and Renamed DefaultContext to Context

huangpf committed:
Revert Versions

ogail committed:
Merge pull request #1431 from hovsepm/AutoRest

Moved TokenCache to Context and Renamed DefaultContext to Context

markcowl committed:
Merge pull request #1429 from huangpf/autorest

AzureRT S54 PR #1 - AutoRest Project & Versions Update

Commit:
Fixed null check for TokenCache.

Commit:
Merge pull request #1434 from hovsepm/AutoRest

Fixed null check for TokenCache.

Commit:
Added support for Context without subscriptionId.

Commit:
Merge pull request #1435 from hovsepm/AutoRest

Added support for Context without subscriptionId.

Commit:
Fix hard-coded SP resource uri issue

markcowl committed:
Merge pull request #1439 from ogail/AutoRest

Fix hard-coded SP resource uri issue

Commit:
Sync from autorest

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

markcowl committed:
Fix exception message in ARM mode [#103697132] and hard coded resource id in authentication [#103425146]

markcowl committed:
Removing unnecessary helper

stankovski committed:
Merge pull request #1448 from markcowl/authupdate

Fix authentication exception messages to include correct cmdlet names

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
Rev the version to include ADLA and ADLS

bitsforthought committed:
Search SDK: Cherry picking latest changes from master to AutoRest

This includes two changes: Fix for JsonConvert issue, and support for
Microsoft analyzers.

stankovski committed:
Merge pull request #1462 from brjohnstmsft/searchmastermerge-pr

Search SDK: Cherry picking latest changes from master to AutoRest

jianghaolu committed:
Add delegating handler exposure to ClientFactory

jianghaolu committed:
Change list to dictionary on key delegating handler's type

jianghaolu committed:
Address CR feedback

jianghaolu committed:
Merge pull request #1465 from jianghaolu/handlers

Add delegating handler exposure to ClientFactory

markcowl committed:
Make client factory use only cloneable handlers

markcowl committed:
Updating version number

markcowl committed:
Ensuring that handlers are added in order

markcowl committed:
Merge pull request #1472 from markcowl/handlers

Force handlers to implement cloneable

Commit:
Sync from latest Azure AutoRest

Commit:
Rev the package to one ahead of the "official"

This allows for me to install and use my private package for now.

x10shun committed:
Latest changes

huangpf committed:
Merge pull request #112 from Azure/AutoRest

Auto rest

x10shun committed:
Fix metrics tests

x10shun committed:
Disable metrics tests in playback mode

x10shun committed:
Update test session records

x10shun committed:
Merge remote-tracking branch 'upstream/AutoRest' into AutoRest

Commit:
Add DataLake SDK packages

Adding DataLake sdk elements for store and analytics.

Commit:
Missed a store rename

Updated to get the store rename

huangpf committed:
Update

x10shun committed:
Disabling clone test. Needs further investigation

Commit:
Updating to use the latest and greatest test frame

Fixing code to remove stylecop complaining comments and update the tests
+ test framework version.

huangpf committed:
Update Version

bitsforthought committed:
Search SDK: Fixing typo in SuggestResult summary doc

Commit:
Somehow a typo ended up in here.

Commit:
Merge pull request #1484 from brjohnstmsft/suggestdocar-pr

Search SDK: Fixing typo in SuggestResult summary doc

stankovski committed:
Merge pull request #1476 from naveedaz/AutoRest

Fixed metrics APIs

markcowl committed:
Adding back UserAgents OM for Visual Studio

huangpf committed:
Merge pull request #119 from Azure/AutoRest

Auto rest

markcowl committed:
Update Common.Authentication version to 1.3.4.0 and add back AddUSerAgent functionality for VS

markcowl committed:
Updating version indicators

markcowl committed:
Adding back tests

Commit:
More incorrect spellings of Tenant

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
URI change and DCRs

change URIs to the official data lake ones
change resource namespaces to the official ones
DCR to improve performance of the multi-part uploader through client:
1. Increase segment size to 256mb always
2. Make thread count a product of the number of segments.

Commit:
Update package versions, manual fixes

Commit:
fixes for concat path naming

Commit:
Update GUIDs and API version

Commit:
Renames and fixes for tests

Disabling job test until job functionality for new accounts is working
again.
Rename datalake -> datalakestore
Add new tests for DataLakeStore and DataLakeStoreFileSystem
Remove broken job test results.

markcowl committed:
Merge pull request #1489 from markcowl/useragents2

Adding back AddUserAgent methods for Visual Studio

Commit:
sync with latest

stankovski committed:
Building against latest client runtime

huangpf committed:
Update AssemblyInfo.cs

Commit:
Fixing up namespaces and default namespaces

and update the test.

stankovski committed:
Merge pull request #1494 from stankovski/AutoRest

Building Test Framework against latest client runtime

Commit:
update the API version for filesystem

Commit:
Update the tests to use new API version

yugangw-msft committed:
Merge pull request #1480 from huangpf/autorest

AzureRT - Update and use autorest 0.11.0-Nightly20150911

Commit:
add direct access apis and job error details

Commit:
Fixing session records to always be copied.

Commit:
switch errorMessage back to a string

Commit:
Fix to remove the requirement on the constructor json.

Commit:
Revert "sync with latest"

This reverts commit 6f47726e0374ceca47741054f29db44a19a4ffd6, reversing
changes made to 6512c47edf7ffa1dcde3cb6514bfc1f458b94dc7.

Reverting temporarily to enable a proper build of the PowerShell using my private Authentication package.

Commit:
temporarily update the package version.

Commit:
reverting temp revert

Commit:
switching the assembly version back to 1.3.4

azuresdkci commented:
Can one of the admins verify this patch?

azuresdkci commented:
Can one of the admins verify this patch?

azurecla commented:
Hi __@yanhuang1990__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

Commit:
Reverting back to 2.12 for AD

updating Hyak.Common to 1.0.3 which fixes a bug with 500 errors.

Commit:
Moving everything to the 2.14 AD dll

Apparently the new TestBase requires 2.14 AD dll. This should resolve
build failures.

Commit:
Updating the tests with the latest run with new AD

huangpf committed:
Merge pull request #124 from Azure/AutoRest

Auto rest

stankovski committed:
Renamed folder for DNX work

stankovski committed:
Reverted the folder rename back

stankovski committed:
Fixed nuspec file

stankovski committed:
Bumped up version or ClientRuntime.Azure

stankovski committed:
Upgraded Newtonsoft.Json dependency to 6.0.8 due to missing target from 6.0.4

Commit:
Concat, test and bug fixes

Added deletesourcedirectory option for concat
Fixed and added more filesystem tests
Fixed nextLink bug

Commit:
One more test update to get the constructor passing

Commit:
missed package rev

stankovski committed:
Fixed compile setting

markcowl commented:
@yanhuang1990 Can you please sign the CLA so we can look at this PR?  Also, it needs to be updated to fix merge conflicts.

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
Update tests and fixes for DataLake Analytic

missed some namespace changes in DataLake.ANalytics. Fixing that now.

stankovski committed:
Moved to DOTNET target

Commit:
One more temporary revert due to final dns name

Updating Data Lake DNS name and need to revert this auth change to
ensure powershell keeps working.

Commit:
Rev the auth package

include new namespace DNS change.

Commit:
Re-Revert to get back to same state as parent

Commit:
Update nuspec data as well as the FileSystem tests.

Commit:
Fix tests to use the test utilities to generate ids

This should fix the json so that file and folder names are statically
defined in the recordings.

Commit:
Fix tests, update test.framework

Update the test framework and update the tests to use the new framework.
Enable the job tests now that the job functionality is online.

Commit:
Update authorization tests with new AD module

Commit:
Fixup the tests for Authorization

stankovski committed:
Updated package version and removed BCL dependency

Commit:
Attempt to streamline logic for uploader

this enables the uploader to throw more quickly if there is a potential
timeout failure in the service.

stankovski committed:
Regenerated client and re-recorded tests

stankovski committed:
Updated Storage client using latest AutoRest and added dnx target

Commit:
Updates to add robustness retry logic

This gives some more retry logic surrounding our calls.

Commit:
Rev package version

stankovski committed:
Updated TestFramework and Common.Authentication to use latest AutoRest client runtime

stankovski committed:
Regenerated Resource and Storage using latest AutoRest

Commit:
Set the per request timeout to 30 seconds

In the future we should have this be configurable. For now, if a request
to the store takes longer than 30 seconds to return, its a dead request.

Commit:
Revving the package version "backwards"

bringing the package version back in line with the ordering being used
in public repos.

stankovski committed:
Regenerated Compute and Network using latest AutoRest

stankovski committed:
Regenerated WebSites client using latest AutoRest

Commit:
Update to latest logic (still doesn't work)

stankovski committed:
Fixed Common.Authentication project

stankovski committed:
Fixed compute tests

stankovski committed:
Fixed Common.Authentication.Tests project

stankovski committed:
Updated Schedule and LogicManagement to use new AutoRest client and ClientRuntime

markcowl committed:
Update AutoRest test framework to depend on http recorder

stankovski committed:
Merge pull request #1502 from stankovski/AutoRest

Added DNX support to ResourceManagement package

markcowl committed:
removing autorestore

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

markcowl committed:
Merging with upstream [#106506538]

Commit:
Test Updates

1. Add DataLake.sln that includes all DataLake* csproj files (for ease
of testing)
2. Update all test json to ensure everything is still passing
3. Update DataLakeStoreFileSystem generated code to have AutoRedirect
always disabled. THIS IS A TEMPORARY SOLUTION. Long term solution is to
move to Swagger/AutoRest functionality which will allow for more
flexibility. As a result of this being temporary, we are making the
minimal change (removal of old logic to switch out http client only).

markcowl committed:
Fixing merge issue in csrpoj and responding to feedback

markcowl committed:
Updating test framework references for libraries

markcowl committed:
Removing references to older test environment property misspellings

markcowl committed:
Adjusting to changes in recording classes

markcowl committed:
Updating RBAC tests to use new test framework

markcowl committed:
Fixing playback issues for authorization tests

markcowl committed:
Merge pull request #1520 from markcowl/httprecorder

Update test framework to depend on Httprecorder

Commit:
Sync with latest from AutoRest

x10shun committed:
Removed webhosting plans API. Bumped version.

Commit:
Fixes to create and cleanup folders during test

These two tests require file system access to actually test the metadata
generation functionality. Updating this test to utilize the current
working directory and cleanup when done.

markcowl committed:
Merge pull request #1522 from naveedaz/AutoRest

Removed webhosting plans API. Bumped version.

huangpf committed:
Merge pull request #138 from Azure/AutoRest

Auto rest

Commit:
Reverting a lot of changes

Reverting changes to authorization and keyvault and packages.config

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

stankovski committed:
Regenerated ARM and Storage clients

stankovski committed:
Regenerated Compute and Network clients

stankovski committed:
Merge pull request #1525 from stankovski/AutoRest

Regenerated AutoRest clients

Commit:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into AutoRest

Commit:
Addressing final comments and removing private text

Removing "DO NOT SHIP"
Reverting KeyVaultTestFixture change.

huangpf committed:
Merge pull request #140 from Azure/AutoRest

Auto rest

markcowl committed:
Merge pull request #1475 from begoldsm/AutoRest

Adding DataLake endpoints

huangpf committed:
Merge pull request #141 from Azure/AutoRest

Auto rest

Commit:
Added Certificate based authentication for SP
- moved markcowl's code from his fork which was based on Master to AutoRest branch

Commit:
Merged PR #1527 to the current changes

Commit:
Referenced the latest Microsoft.Rest.ClientRuntime.Azure.Authentication

Commit:
Added CertificateApplicationCredentialProvider

Commit:
Setting Datalake environment to null for US Gov.

Commit:
Addressing feedback

huangpf committed:
VMSS

Commit:
Merge pull request #1528 from hovsepm/AutoRest

[#106317732] Added Certificate based authentication for SP

huangpf committed:
Update

huangpf committed:
Fix

huangpf committed:
Merge pull request #148 from Azure/AutoRest

Auto rest

huangpf committed:
Merge pull request #149 from huangpf/autorest

Autorest

stankovski committed:
Regenerated Storage client using latest AutoRest

stankovski committed:
Fixed test related to Storage change

huangpf committed:
Update

huangpf committed:
Update & Re-record Tests

huangpf committed:
Add Tests for BootDiagnostics, DiskSize & DiskEncryption

huangpf committed:
Recover & re-record 1 VMSS test

huangpf committed:
Recover VMSS Operational Tests

huangpf committed:
Recover VMSS Update Tests

huangpf committed:
Recover VMSSVM Tests

huangpf committed:
Fix StatusesSummary Issue

stankovski committed:
Merge pull request #1530 from huangpf/autorest2

AzureRT PR - AutoRest VMSS SDK APIs

stankovski committed:
Merge branch 'AutoRest' of https://github.com/WindowsAzure/azure-sdk-for-net into AutoRest

stankovski committed:
Regenerated Compute client using latest AutoRest

stankovski committed:
Merge pull request #1532 from stankovski/AutoRest

Regenerated Storage and Compute client using latest AutoRest

huangpf committed:
Re-record VMSS Tests

huangpf committed:
Merge branch 'AutoRest' of https://github.com/Azure/azure-sdk-for-net into autorest2

huangpf committed:
Update ComputeManagementClient.json

huangpf committed:
Update Tests

huangpf committed:
+Capture & User Image Creation Test

stankovski committed:
Merge pull request #1538 from huangpf/autorest2

AzureRT PR - Re-record VMSS Tests
